test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -200,8 +200,8 @@ private static void testMemorySwapLimitSanity() throws Exception {
             .shouldMatch("Memory Limit is:.*" + expectedTraceValue)
             // Either for cgroup v1: a_1) same as memory limit, or b_1) -2 on systems with swapaccount=0
             // Either for cgroup v2: a_2) 0, or b_2) -2 on systems with swapaccount=0
-            .shouldMatch("Memory and Swap Limit is:.*(" + expectedTraceValue + "|-2|0)")
-            .shouldNotMatch("Memory and Swap Limit is:.*" + neg2InUnsignedLong);
+            .shouldMatch("(Memory and )?Swap Limit is:.*(" + expectedTraceValue + "|-2|0)")
+            .shouldNotMatch("(Memory and )?Swap Limit is:.*" + neg2InUnsignedLong);
```

Comment:
This is a side-effect of reading the swap limit on cg v2 which only prints `Swap Limit is:` (fixing a FIXME along the way in the code), since it's only reading the swap value in that case. Note that for cg v2 the interface files for swap limit and memory limit are different files and for cg v2, only the swap limit is there (contrary to the cg v1 interface file which contains the memory and swap limit). Therefore I've done an optional match in the test.